wikimedia/wikipedia-ios
# Update scripts/localization

**Phabricator:** 
N/A

### Notes
Some [helpful comments](https://github.com/wikimedia/wikipedia-ios/pull/4462) were added to our localization script, however we will not see them until we update our executable script in `scripts/localization`. I tried following the steps in our localization documentation, but they didn't work on my machine. I exported the binary by going through the Archive > Distribute flow in Xcode, and updated the docs to match.

This script runs on two of my machines, but we should confirm that it runs on others as well.

### Test Steps
1. In `main`, cd into the repo directory.
2. From there, run `./scripts/localization {path to repo directory}`. Note how the script runs, but no additional comments from #4462 display.
3. Now switch to this PR branch and repeat steps 1-2. You should see additional comments from #4462 printed in the console.
